redirector.k8s.io/deployment.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: redirector-k8s-io
+  namespace: redirector-k8s-io-prod
+  labels:
+    app: redirector-k8s-io
+    version: dirty
+spec:
+  revisionHistoryLimit: 2
+  replicas: 2
+  selector:
+    matchLabels:
+      app: redirector-k8s-io
+      component: txtdirect
+  template:
+    metadata:
+      labels:
+        app: redirector-k8s-io
+        component: txtdirect
+    spec:
+      securityContext:
+        # Assign files to non-root supplementary GID
+        fsGroup: 1337
+        runAsNonRoot: true
+      containers:
+      - name: txtdirect
+        image: gcr.io/txtdirect-223710/txtdirect:dev-00984e2
```

Comment:
we'll have to get a binary to upload to k8s.gcr.io.  We can grant you a staging repo and use the promoter

Reply:
+1

Reply:
PR: https://github.com/kubernetes/k8s.io/pull/354